llvm/docs/ReleaseNotes.md

```diff
@@ -191,6 +191,10 @@ Changes to LLDB
 * The `show-progress` setting, which became a NOOP with the introduction of the
   statusline, now defaults to off and controls using OSC escape codes to show a
   native progress bar in supporting terminals like Ghostty and ConEmu.
+* The default PDB reader on Windows was changed from DIA to native, which uses 
+  LLVM's PDB and CodeView support. You can switch back to the DIA reader with
+  `settings set plugin.symbol-file.pdb.reader dia`. Note that support for the
```

Comment:
Should we mention the `LLDB_USE_NATIVE_PDB_READER` here? Looks like users can turn native PDB off by setting that envvar off explicitly?

Or did we decide we don't want to encourage people to use that?

Reply:
I wouldn't mind either way. The setting is not in any release yet. Before, you had to use the environment variable.